RealmModelGenerator/EntitiesView.swift

```diff
@@ -35,11 +50,30 @@ class EntitiesView: NibDesignableView, NSTableViewDelegate, NSTableViewDataSourc
             cell.title = "Entity"
             return cell
         }
-        cell.title = "\(row)"
-        return cell
+        return self.dataSource?.entitiesView(tableView, viewForTableColmn: tableColumn, row:row)
+    }
+    
+    func tableViewSelectionDidChange(notification: NSNotification) {
+        let tableView = notification.object as! NSTableView
+        selectedIndex = tableView.selectedRow
+        if let index = selectedIndex {
+            self.delegate?.entitiesView(self, selectionDidChange: index)
+        }
+    }
+    
+    @IBAction func addButton(_: AnyObject) {
+        self.delegate!.addEntityInEntitiesView(self)
+        reloadData()
+    }
+    
+    @IBAction func removeButton(_: AnyObject) {
+        if let index = selectedIndex {
+            self.delegate?.removeEntityInEntitiesView(self, index: index)
+            reloadData()
+        }
     }
     
-    override func prepareForInterfaceBuilder() {
-        super.prepareForInterfaceBuilder()
+    func reloadData() {
+        tableView.reloadData()
```

Comment:
if there is currently no selected row then we should probably have the minus button disabled